src/OrbitDB.js

```diff
@@ -28,26 +28,26 @@ let databaseTypes = {
   'keyvalue': KeyValueStore,
 }
 
-class OrbitDB {
-  constructor(ipfs, directory, options = {}) {
+  class OrbitDB {
+  constructor(ipfs, identity, options = {}) {
     this._ipfs = ipfs
     this.id = options.peerId
-    this.identity = options.identity
+    this.identity = identity
```

Comment:
Should we throw an error here if identity wasn't provided since it's required (I know we don't do that for ipfs yet either, but would prolly be good to throw to guide the user to call the constructor correctly)?

Reply:
If we do ^, could also add the same check for IPFS while we're at it.